### DISCUSSION TITLE
Where are templates stored?

### DISCUSSION THREAD

**Discussion by MikaelAnderssonWigander** [jmstoolbox/jmstoolbox#185] [Q&A] [answered]
Hi

When exporting configuration none of my templates where exported to the zip.
Therefor I wonder where the template files are located so I can manually back them up?

**Comment by titou10titou10** [comment, marked as answer]
Everything is in the home folder of the user running JTB eg  `C:\Users\<user>\.jtb\JMSToolBox` or `<user home>\.jtb\JMSToolBox`
Templates are in `C:\Users\<user>\.jtb\JMSToolBox\Templates`

Now the fact that templates are not exported looks like a bug..
